hack/update-swagger-spec.sh

```diff
@@ -18,7 +18,7 @@ function cleanup()
     rm -rf "${TEMP_DIR}"
 
     if [ $out -ne 0 ]; then
-        echo "[FAIL] !!!!! Generate Failed !!!!"
+        echo -e "\033[0;31m [FAIL] !!!!! Generate Failed !!!!\033[0m"
```

Comment:
Colors just aren't important in this flow, and they make readability of the script harder.